lib/app-config.rb

```diff
@@ -22,7 +22,7 @@ class Config
     # Returns the new config.
     def initialize(options)
       @options = options
-      yaml = YAML.load(ERB.new(IO.read @options.config_file).result)
+      yaml = YAML.load(ERB.new(IO.read @options.config_file).result, aliases: true)
```

Comment:
How come you didn't swap to `safe_load` like I did in https://github.com/jalada/yappconfig/pull/7/files?